onnxruntime/core/providers/cpu/math/element_wise_ops.cc

```diff
@@ -1034,4 +1037,226 @@ Status Erf<float>::Compute(OpKernelContext* context) const {
   return Status::OK();
 }
 
+class Mod final : public OpKernel {
```

Comment:
nit pick : create a dedicated file for this op since its implementation is long?